compiler/testData/codegen/bytecodeText/builtinFunctions/removeAt.kt

```diff
@@ -1,3 +1,4 @@
+// IGNORE_BACKEND: JVM_IR
```

Comment:
Maybe we can encapsulate the difference in bytecode text in JVM/JVM_IR into `JVM_TEMPLATES` / `JVM_IR_TEMPLATES` directives in this test?

Reply:
Done.

I forgot about this test - the reason for the difference is that the new bridge methods for generic special  bridges end up calling the instantiated version of `remove(I)` instead of `removeAt(I)`. This is necessary for type-safe special bridges - since there is additional logic in the wrapper - but unnecessary for methods which were merely renamed. We might want to match the behavior of the old backend in this case (apart from some missing bridges in this test).